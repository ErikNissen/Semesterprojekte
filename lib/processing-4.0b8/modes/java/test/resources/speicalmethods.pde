version https://git-lfs.github.com/spec/v1
oid sha256:38c73fc53bdb64609f664ec0ce3d79cad8573dbee275a7388f3836da78fe8140
size 342
