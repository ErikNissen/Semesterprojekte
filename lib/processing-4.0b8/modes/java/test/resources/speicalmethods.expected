version https://git-lfs.github.com/spec/v1
oid sha256:30e41cb928194a2370af0ea8d71122c736f6f3bec84b31482adc6d83e7f3f5b0
size 1093
